scss/_patterns_switch.scss

```
@@ -1,8 +1,9 @@
 @import 'settings';
-$knob-size: $sp-unit * 3;
+$knob-size: $sp-unit;
```

Sorry to bother about this again, but you are doing a lot of multiplications when this is too small.

I'd suggest using 
```suggestion
$knob-size: $sp-unit * 2;
```

Which will be 1rem, and expected knob-size.

Then in all places where you need the knob-size you can just use this variable directly. And for the width of the switch multiply it by 2.